packages/transformers/src/tests/compile-ts-fragment.ts

```diff
@@ -0,0 +1,153 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import * as ts from 'typescript';
+
+/**
+ * Filesystem caching for the TypeScript CompilerHost implementation used in
+ * `compileTsFragment`. Shared across multiple invocations to significantly
+ * speed up the loading of common files (e.g. TypeScript core lib d.ts files).
+ * Assumes files on disk never change.
```

Comment:
Consider calling this out a bit more, I almost missed it. Maybe put it on its own line and do:

```
**Important Note**: Assumes files on disk never change
```

Maybe name it something that indicates that its caching makes assumptions, like `OptimisticallyImmutableCompilerHostCache` or `OverlyOptimisticCompilerHostCache`

Reply:
FYI just realized when I re-organized the commits I forgot to delete the other version of this file. So, this is not actually a new file, it just moved from `localize-tools` (though your comment is still valid). Added the delete to the newest commit since I don't want to invalidate the commits with comments anymore.

Reply:
Added an "Important!" comment with explanation. Also just in case this wasn't clear, note this is in the test directory and used to speed up loading standard TS libs for unit tests.